common/djangoapps/util/enterprise_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utilities to communicate with the Enterprise service.
```

Comment:
I think it's going to confuse people to have two modules (`enterprise_api` and `enterprise_helpers`) so it would be good to combine them into a single module if possible.